### PR TITLE
 b2c category nodes endpoints tests

### PR DESCRIPTION
**Pull request by himanshu-pal-metacube**
CC-16151   b2c category nodes endpoints

### CODE REVIEW

File: tests/api/b2c/glue/category_endpoints/category-nodes/negative.robot [hunk 1]
```diff
@@ -0,0 +1,26 @@
+*** Settings ***
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/category_endpoints/category-nodes/negative.robot:1]: please follow naming convention - folder name should be with _ and not with -

File: tests/api/b2c/glue/category_endpoints/category-nodes/positive.robot [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][children]    name
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][children]    isActive
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][children]    order
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][children]    url
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][children]    children
+    And Each Array Element Of Array In Response Should Contain Property:    [data][attributes][children]    parents
+    And Response body has correct self link internal
+
+
+Get_category_node_is_leaf_by_id
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/category_endpoints/category-nodes/positive.robot:47]: please rename here and in b2b tests to 'Get_category_node_that_has_only_parents_by_id'

File: tests/api/b2c/glue/category_endpoints/category-nodes/negative.robot (hunk 1 above)
Comment (himanshu-pal-metacube) [reply tests/api/b2c/glue/category_endpoints/category-nodes/negative.robot]: Done

File: tests/api/b2c/glue/category_endpoints/category-nodes/positive.robot (hunk 2 above)
Comment (himanshu-pal-metacube) [reply tests/api/b2c/glue/category_endpoints/category-nodes/positive.robot]: Done

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**